todo_app/my_ansible_playbook.yml

```diff
@@ -0,0 +1,50 @@
+- name: install and run to-do app
+  hosts: webservers
+  remote_user: ec2-user
+  vars_files:
+  - secrets.yml
```

Comment:
It's worth noting that as long as this has been encrypted, you're welcome to check in the secrets to your repo - just don't include the password!